README.md

```diff
@@ -66,6 +74,8 @@ Post installation steps:
 ```bash
 sudo groupadd docker
 sudo usermod -aG docker $USER
+sudo chown -r centos:centos /home/centos/
```

Comment:
```suggestion
# In case you have any permission problems make sure your user (centos) has the correct rights 
# You can check this with:
ls -l

# change ownership only if necessary
sudo chown -r centos:centos /home/centos/
```